f/connectors/globalforestwatch/gfw_alerts.py

```diff
@@ -0,0 +1,161 @@
+# requirements:
+# psycopg2-binary
+# requests~=2.32
+
+import json
```

Comment:
I just barely skimmed this file, assuming the brunt of it has been reviewed previously in another repo.